tests/robot-tests/tests/admin_and_public/bau/publish_methodology.robot

```diff
@@ -111,35 +111,25 @@ Update the methodology amendment's content
     user changes accordion section title    1    New and Updated Title    ${METHODOLOGY_CONTENT_EDITABLE_ACCORDION}
 
 Add a note describing the amendment
-    # TODO EES-2302 / EES-2305 - Replace me when methodology notes can be created via content page
-    ${methodologyVersionId}=    get methodology version id from url
-    set suite variable    ${AMENDMENT_METHODOLOGY_VERSION_ID}    ${methodologyVersionId}
-    user creates methodology note via api
-    ...    ${AMENDMENT_METHODOLOGY_VERSION_ID}
+    user adds note to methodology
     ...    Latest note
-    ...    2021-09-16T23:00:00Z
 
 Add and remove another note describing the amendment
-    # TODO EES-2302 / EES-2305 - Replace me when methodology notes can be created via content page
-    ${noteIdToRemove}=    user creates methodology note via api
-    ...    ${AMENDMENT_METHODOLOGY_VERSION_ID}
+    user adds note to methodology
     ...    Note which should be deleted
-    ...    2021-09-16T23:00:00Z
-    user removes methodology note via api
-    ...    ${AMENDMENT_METHODOLOGY_VERSION_ID}
-    ...    ${noteIdToRemove}
+    user removes methodology note
+    ...    Note which should be deleted
+    ...    css:#methodologyNotes li:nth-of-type(1)
 
 Add and update another note describing the amendment
-    # TODO EES-2302 / EES-2305 - Replace me when methodology notes can be created via content page
-    ${noteIdToUpdate}=    user creates methodology note via api
-    ...    ${AMENDMENT_METHODOLOGY_VERSION_ID}
+    user adds note to methodology
     ...    Note which should be updated
-    ...    2021-09-16T23:00:00Z
-    user updates methodology note via api
-    ...    ${AMENDMENT_METHODOLOGY_VERSION_ID}
-    ...    ${noteIdToUpdate}
+    user edits methodology note
     ...    Earliest note
-    ...    2021-03-01T00:00:00Z
+    ...    01
+    ...    03
+    ...    2021
+    ...    css:#methodologyNotes li:nth-of-type(1)
```

Comment:
I'd switch to passing the original note text to `user edits methodology note` and then have it handle clicking the correct Edit note button. Then the tests will be more robust if someone adds more methodology notes and you don't have to worry about selectors when writing tests.